pkg/api/validation/validation.go

```diff
@@ -183,6 +183,13 @@ func ValidateEndpointsName(name string, prefix bool) (bool, string) {
 	return NameIsDNSSubdomain(name, prefix)
 }
 
+// ValidateEventName can be used to check whether the given event name is valid.
+// Prefix indicates this name will be used as part of generation, in which case
+// trailing dashes are allowed.
+func ValidateEventName(name string, prefix bool) (bool, string) {
+	return NameIsDNSSubdomain(name, prefix)
```

Comment:
I also don't know what events are being created today to know if this will break existing clients